test.js

```diff
@@ -2,6 +2,7 @@ import test from 'ava';
 import m from './';
 
 test(t => {
+	t.throws(() => m('foo'), 'Expected `year` to be of type `Date` or `number`, got `string`');
```

Comment:
Can you just check for a TypeError here? I don't like having to duplicate the error message in multiple places.